app/models/tag.rb

```diff
@@ -101,6 +102,13 @@ def bust_cache
     Tags::BustCacheWorker.perform_async(name)
   end
 
+  def bust_articles_cache
+    article_ids = Article.tagged_with(name).order(created_at: :desc).limit(100).ids
```

Comment:
this can probably be simplified with the following:

```suggestion
    article_ids = articles.published.order(created_at: :desc).limit(100).ids
```

I also added the `published` scope, because the goal was to bust published articles, not drafts

Reply:
Oops I missed that. Will modify it :+1: